### ISSUE TITLE
Ecommerce Replacement

### ISSUE THREAD

**Issue by BbrSofiane** [openedx/build-test-release-wg#143]
This issue is for the BTR group to have a reminder to track the progress of [the ecommerce replacement](https://github.com/openedx/platform-roadmap/issues/83) work.

Ben Holt @inventhouse (2U) and Feanil Patel @feanil (tCRIL) are the lead on this.

**Comment by nedbat** [comment]
I don't think this will have any concrete effect on Nutmeg because of the long timelines involved.

**Comment by nedbat** [comment]
Deprecation announcement: https://discuss.openedx.org/t/deprecation-removal-ecommerce-service-depr-22/6839

Questions:
- [ ] What does April 1 deprecation mean?
- [ ] Does ecommerce-coordinator have a role in the Nutmeg timeframe?

**Comment by feanil** [comment]
Hopefully the response on the discussion makes it more clear, for the second question, I'm not sure but I think asking on that thread would be the correct next step.

**Comment by arbrandes** [comment]
Closing this.  From [the thread](https://discuss.openedx.org/t/deprecation-removal-ecommerce-service-depr-22/6839/2?u=arbrandes) it's pretty clear what the deprecation means:  "Very early directional announcement and is meant to communicate that the ecommerce service is going to be put into maintenance mode.  There will be more communication when it is ready to be removed."

From the [repo](https://github.com/edx/commerce-coordinator) and other conversation, it's clear that commerce coordinator is still in an experimental state, and thus not elligible for inclusion in Nutmeg.

**Comment by arbrandes** [comment]
Forgot the issue is more general than just commerce coordinator.  Reopening.
